packages/react-components/lib/doors/door-panel.tsx

```diff
@@ -30,39 +30,50 @@ export interface DoorInfoProps {
 const useStyles = makeStyles((theme) => ({
   container: {
     margin: theme.spacing(1),
+    backgroundColor: theme.palette.primary.main,
   },
   buttonBar: {
     display: 'flex',
     justifyContent: 'flex-end',
-    borderRadius: '0px',
-    backgroundColor: theme.palette.primary.main,
+    borderRadius: 0,
+    backgroundColor: theme.palette.primary.light,
   },
   grid: {
-    padding: '1rem',
+    padding: theme.spacing(1),
   },
   doorLabelOpen: {
     backgroundColor: theme.palette.success.main,
+    color: theme.palette.getContrastText(theme.palette.success.main),
   },
   doorLabelClosed: {
     backgroundColor: theme.palette.error.main,
+    color: theme.palette.getContrastText(theme.palette.error.main),
   },
   doorLabelMoving: {
     backgroundColor: theme.palette.warning.main,
+    color: theme.palette.getContrastText(theme.palette.warning.main),
   },
   cellPaper: {
-    padding: '0.5rem',
-    backgroundColor: theme.palette.info.light,
+    padding: theme.spacing(2),
+    backgroundColor: theme.palette.primary.light,
+    color: theme.palette.getContrastText(theme.palette.primary.light),
   },
   itemIcon: {
     color: theme.palette.getContrastText(theme.palette.primary.main),
   },
   buttonGroup: {
+    marginTop: theme.spacing(1),
     display: 'flex',
     justifyContent: 'center',
   },
+  buttons: {
+    backgroundColor: theme.palette.primary.main,
+    color: theme.palette.getContrastText(theme.palette.primary.main),
```

Comment:
```suggestion
    color: theme.palette.primary.main.contrastText,
```

I think this way is slight more performant.

Same goes for all the other places uses `getContrastText`.

Reply:
sorry I must have missed this out

Reply:
oh i've been confusing the infrastructure tab changes with these because they do overlap, but yep will update the getContrastTexts!